src/SingleCellAnnotator.imjoy.html

```diff
@@ -468,6 +468,9 @@
         }
         this.stopMaskAnnotation();
       };
+      if(this.current_image_data.location_changed){
+        await this.uploadAnnotation();
```

Comment:
When will we hit this case? Since the annotation buttons will trigger upload itself, I cannot think of a case that we need to upload when we click back. No?

Reply:
think about precise annotation, if you miss labeling one protein or remove one labeled protein, under this circumstance, you will make change of the labeled location. after the change, when click Back, it should trigger upload annotation

Reply:
You are right! All good then.